### PR TITLE
SDNTB-164 Untranslated strings

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: scripts/superdesk-archive/views/item-state.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<span class="state-label state-{{state}}">{{state|removeLodash}}</span>
+<span class="state-label state-{{state}}">{{state|translate|removeLodash}}</span>
 <span class="state-label" ng-if="embargo" ng-class="{state_embargo: embargo}">embargo</span>
```
Comment (mdhaman) [inline scripts/superdesk-archive/views/item-state.html]: translate missing.

**Comment by mdhaman** [comment]
:+1: looks good

**Comment by ioanpocol** [comment]
:+1:  (except small comment)

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**